akmodsposttrans@.service

```diff
@@ -0,0 +1,7 @@
+[Unit]
+Description=Building and installing kmods for kernel %i
+
+[Service]
+Type=simple
+RemainAfterExit=yes
+ExecStart=/usr/bin/systemd-inhibit --who=akmods --why="Building kernel modules" /usr/sbin/akmods --from-kernel-posttrans --kernels %i
```

Comment:
Well, you need to test and say for sure.
BTW, it would be fine you rebase your commit on top current master and drop the akmods-shutdown removal (please don't enable it). It's unclean to have several unrelated changes into the same PR

I'd like to test akmodposttrans without systemd-inihibit.
Also I think this should be better named, such as akmods-for-kernel@ as it better suggest that the kernel parameter is what is required next.